packages/documentation-v7/src/stories/components/textarea/textarea.stories.ts

```diff
@@ -0,0 +1,262 @@
+import type { Args, Meta, StoryContext, StoryObj } from '@storybook/web-components';
+import { html, nothing } from 'lit';
+import { useArgs } from '@storybook/preview-api';
```

Comment:
We used updateArgs and useState before to sync the value of the textarea between the docs mode and the stories view.
I think we do not need this anymore.

```suggestion
```